addons/mail/static/src/models/chatter.js

```diff
@@ -385,6 +385,15 @@ registerModel({
         isShowingAttachmentsLoading: attr({
             default: false,
         }),
+        /**
+         * Determines whether the view should reload after file changed in this chatter,
+         * such as from a file upload.
+         */
+        reloadFromFileChanged: attr({
```

Comment:
`reloadFromFileChanged` ➡️ `shouldReloadParent`?

I would like to see words like "should" appear to suggest that this is a boolean. Also, I think it's important to make it clear that what needs to be reloaded is the parent, not the chatter itself. I don't understand the "fromFileChanged" part, where does it come from and why is it necessary to be so specific?

Reply:
Without `fromFileChanged`, context of when reload should maybe happen is lost.
So it's unclear when this condition might have an impact.

Ok for remarks on `should` and `parent`.
Note that `parent` is not clearly conceptualized in the code. I would even say that if we do it properly, it shouldn't be named `parent`. But I will put `parent` just for consistency and clarity is not just chatter itself.

Reply:
Change done

Reply:
don't forget to reorder the field alphabetically :)

Reply:
done!